components/rating/rating.css

```diff
@@ -0,0 +1,17 @@
+.rating{
+    margin: 0.5em;
+    font-size:1.5em;
+    direction: rtl;
+    text-align: left;
+}
+.rating > span {
```

Comment:
add 1 line space after each css block for better readability

Reply:
Ok, will take care of it.